ocs_ci/utility/kubevirt_vm.py

```
@@ -232,8 +232,8 @@ def unpause_kubevirt_vms(self, vms, wait=True):
         """
         for vm in vms:
             vm_name = get_vm_name(vm)
-            logger.info(f"Pausing the VM {vm_name}")
-            cmd = f"pause {vm_name}"
+            logger.info(f"UnPausing the VM {vm_name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)